mypy/build.py

```diff
@@ -203,6 +203,9 @@ def default_data_dir(bin_dir: str) -> str:
     # TODO fix this logic
     if not bin_dir:
         # Default to directory containing this file's parent.
+        if os.path.dirname(os.path.dirname(__file__)).endswith('.egg'):
+            return os.path.join(os.path.dirname(os.path.dirname(__file__)),
+                                'lib', 'mypy')
         return os.path.dirname(os.path.dirname(__file__))
```

Comment:
Let's factor out `os.path.dirname(os.path.dirname(__file__))` into a local variable so we don't have to repeat it.